src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Nodes/TreeNode.cs

```diff
@@ -235,42 +237,43 @@ public NodeInfo ToNodeInfo()
                 NodeStatus = this.NodeStatus,
                 NodeSubType = this.NodeSubType,
                 ErrorMessage = this.ErrorMessage,
-                ObjectType = this.NodeTypeId.ToString()
+                ObjectType = this.NodeTypeId.ToString(),
+                FilterableProperties = this.FilterProperties
             };
         }
 
         /// <summary>
         /// Expands this node and returns its children
         /// </summary>
         /// <returns>Children as an IList. This is the raw children collection, not a copy</returns>
-        public IList<TreeNode> Expand(string name, CancellationToken cancellationToken, string? accessToken = null)
+        public IList<TreeNode> Expand(string name, CancellationToken cancellationToken, string? accessToken = null, NodeFilter[]? filters = null)
```

Comment:
>NodeFilter[]? filters [](http://example.com/codeflow?start=116&length=21)

I am curious why you use arrays instead of something like
`IEnumerable<NodeFilter>? filters = null`